variables.tf

```diff
@@ -85,6 +85,9 @@ variable "workers_group_defaults" {
     asg_max_size         = "3"           # Maximum worker capacity in the autoscaling group.
     asg_min_size         = "1"           # Minimum worker capacity in the autoscaling group.
     instance_type        = "m4.large"    # Size of the workers instances.
+    root_volume_size     = "20"          # root volume size of workers instances.
```

Comment:
Should have asked before I merged: Are these values the defaults that worker ASGs received before this change?

Reply:
Not sure. When I start EKS worker as `m5.xlarge`, its root disk size is 20GB, has no chance to test with other instance. 

But if AWS designs the EKS workers and set 20GB with gp2, it should be very reasonable and I also think it works for most case. 

But if you need double check, I can try to start other instance type or if anyone can directly give the data if they already run with other instance types.

Reply:
@brandoconnor 

I can confirm the default root size now: 

https://github.com/awslabs/amazon-eks-ami/blob/master/eks-worker-al2.json#L22

above repo is managed by AWS EKS team and use packer to make worker AMI images.